### PR TITLE
Fixing Android Sample User Handle Leak

### PR DESCRIPTION
**Pull request by MSFT-Heba**
Fixing Android Sample User Handle Leak

### CODE REVIEW

File: MobileSDK/Samples-Native/Sample-Android/build.gradle [hunk 1]
```diff
@@ -26,7 +26,7 @@ allprojects {
             name 'Google'
         }
         maven {
-            url 'file:///C:/git/sdk.samples/MobileSDK/XboxLiveSDK/Android/Maven'
+            url 'file:///C:/repos/samples/xbox-live-samples/MobileSDK/XboxLiveSDK/Android/Maven'
```
Comment (jasonsandlin) [inline MobileSDK/Samples-Native/Sample-Android/build.gradle]: >             [](start = 0, length = 12)

does this need to be relative path?  absolute paths won't work if repro isn't there

Comment (MSFT-Heba) [reply MobileSDK/Samples-Native/Sample-Android/build.gradle]: yeah it needs to be but it's already an absolute path. Planning on fixing it in a new PR (I tried a couple relative path tricks but none of them have worked so far)

Comment (Q800) [reply MobileSDK/Samples-Native/Sample-Android/build.gradle:29]: > > ```
> >         [](start = 0, length = 12)
> > ```
> 
> does this need to be relative path? absolute paths won't work if repro isn't there

**Review by Q800 — COMMENTED** [review]
#105

**Pull request merged**